jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -166,7 +166,11 @@ pipeline {
             steps {
                 script {
                     // Retrieve PROJECT_VER from version-def.sh, e.g, '<major>.<minor>.<patch>-SNAPSHOT'
-                    PROJECT_VER = sh(returnStdout: true, script: "bash $JENKINS_ROOT/version-def.sh | cut -d ',' -f 3 | cut -d ' ' -f 3")
```

Comment:
Change to retrieve PROJECT_VER

The previous way of parsing the bash output by `| cut -d ',' -f 3 ... ` from `version-def.sh` can not work well,  if we change `echo` string in version-def.sh

The previous output from `version-def` is:
`CUDF_VER: 22.12.0-SNAPSHOT, CUDA_CLASSIFIER: cuda11, PROJECT_VER: 22.12.0-SNAPSHOT, SPARK_VER: 3.1.1 ...`

 But `| cut -d ',' -f 3 ... ` does not work if we :

 add `echo $arcg` ahead,  
        --> bash output : `amd64 CUDF_VER: 22.12.0-SNAPSHOT, CUDA_CLASSIFIER: cuda11, PROJECT_VER: 22.12.0-SNAPSHOT, SPARK_VER: 3.1.1 ..` 
        --> PROJECT_VER=`amd64 22.12.0-SNAPSHOT`

or if remove `CUDF_VER: 22.12.0-SNAPSHOT,` from previous output:
        --> bash output : `CUDA_CLASSIFIER: cuda11, PROJECT_VER: 22.12.0-SNAPSHOT, SPARK_VER: 3.1.1 ..` 
        --> PROJECT_VER=`3.1.1`

Reply:
please revert this file change too

Reply:
sure, will update `jenkins/Jenkinsfile-blossom.premerge` in follow up PR